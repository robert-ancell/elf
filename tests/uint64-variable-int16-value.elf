int16 v = 32767
uint64 value = v
print (value)
