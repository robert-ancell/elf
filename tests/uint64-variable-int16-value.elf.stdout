Line 2:
uint64 value = v
       ^^^^^
Variable is of type uint64, but value is of type int16
